xroot/Overlay.lua

```diff
@@ -190,7 +212,7 @@ end
 local function onStartFloatingMenu()
   floatingMenuTimer = nil
   local choices = floatingMenuObject:getFloatingMenuItems()
-  floatingMenuObject = nil
```

Comment:
This doesn't seem to be necessary. Having it here though prevents the "fire" methods I added from working though since the reference is gone.